packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -534,27 +535,23 @@ class AndroidWebViewPermissionRequest extends PlatformWebViewPermissionRequest {
   }
 }
 
-/// A callback interface used by the host application to set the Geolocation permission state for an origin.
-class GeolocationPermissionsRequest {
-  GeolocationPermissionsRequest._(
-      this.origin, android_webview.GeolocationPermissionsCallback callback)
-      : _callback = callback;
+/// A response used by the host application to set the Geolocation permission state for an origin.
+class GeolocationPermissionsResponse {
+  /// [isAllow]: Whether or not the origin should be allowed to use the Geolocation API.
```

Comment:
Why did you prefix these with `is`? Neither reads correctly this way.

Reply:
I have changed it.